parquet-cli/src/main/java/org/apache/parquet/cli/BaseCommand.java

```diff
@@ -305,11 +305,11 @@ protected static ClassLoader loaderForPaths(List<String> paths)
         // TODO: add these to the reader builder
```

Comment:
This file is not 'avro tests'.

Reply:
I've reverted the change. I'll open a new PR.